data/splitContent.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "type": "splitContent",
+    "props": {
+      "children": [
+        [
```

Comment:
I agree that in theory it should be a list of lists but this makes things inconsistent.

Maybe it should be:
```:json

[
  {
    "type": "splitContent",
    "props": {
      "children": [
        {
          "type": "splitArea",
          "props": {
            "children": [
              {
                "type": "text",
                "props": {
                  "variant": "markdown",
                  "value": "Left content"
                }
              }
            ]
          }
        },
        {
          "type": "splitArea",
          "props": {
            "children": [
              {
                "type": "text",
                "props": {
                  "variant": "markdown",
                  "value": "Left content"
                }
              }
            ]
          }
        }
      ]
    }
  }
]
```